libsolidity/ast/Types.cpp

```diff
@@ -573,10 +573,8 @@ namespace
 
 bool isValidShiftAndAmountType(Token _operator, Type const& _shiftAmountType)
```

Comment:
```suggestion
bool isValidShiftAndAmountType(Token, Type const& _shiftAmountType)
```

The build failures in CI are due to warnings about an unused variable here. Not giving it a name would solve that, but even better would be to remove the argument entirely (i.e. also on the call-sites).

Reply:
Thanks for the feedback , will make the required changes.